packages/create-svelte/index.js

```diff
@@ -72,7 +72,7 @@ function write_common_files(cwd, options, name) {
 
 	pkg.dependencies = sort_keys(pkg.dependencies);
 	pkg.devDependencies = sort_keys(pkg.devDependencies);
-	pkg.name = to_valid_package_name(name);
+	pkg.name = name;
```

Comment:
I think the way you implemented this is worse than actual implementation...

now, if you initialize project in existing folder like `some@folder#`, it will throw error instead converting it to correct package name `some-folder`

Reply:
Incorrect, it is impossible to provide an invalid package name as currently implemented -- the folder name is no longer used as the package name; the package name (which is supplied by the user and validated before acceptance) is used as the default folder name. Instead of _converting_ the folder name to a valid (and sometimes really weird) package name, we're now just starting with a guaranteed-to-be-valid package name.

Reply:
you actually just repeated my words about how it's from another input... I just say this is not really good solution and it should be derived from folder name. That's how `npm init -y` behave itself.

Reply:
> now, if you initialize project in existing folder like some@folder#, it will throw error instead converting it to correct package name some-folder even when name of folder is valid.

I was responding to this, which is just plain incorrect. 

As for the rest of your comment, not wanting to add another input is a perfectly valid concern, but not one best discussed here, on a hidden comment, on an unrelated line of code. I'm going to resolve this thread, as that same concern is going to come up in my top-level reply to the last comment on this PR -- feel free to provide your feedback there after I've made it. If you have a better idea on how to support scoped package names (which can't really be derived from a provided path without some _very_ surprising edge cases), I would love to hear them!

Reply:
I look on your code, and I don't see how it could derive package name, because I don't see any derivation logic as is used now.